src/tests/ftest/util/daos_utils.py

```diff
@@ -495,6 +495,14 @@ def container_list_attrs(self, pool, cont, sys_name=None):
         # aa bb
         # attr48
         match = re.findall(r"\n([\S ]+)", self.result.stdout_text)
+        # strip all trailing spaces from new daos cmd output formatting
+        # XXX to be removed when DAOS-7093 will be fixed and API/pydaos
```

Comment:
(pylint-fixme) XXX to be removed when DAOS-7093 will be fixed and API/pydaos

Reply:
What's XXX?